content/en/docs/components/central-dash/profiles.md

```diff
@@ -6,13 +6,21 @@ weight = 20
 
 ## What is a Kubeflow Profile?
 
-A Profile is a Kubernetes CRD introduced by Kubeflow that wraps a Kubernetes Namespace.
+A Kubeflow Profile is a [Kubernetes CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) introduced by Kubeflow that wraps a Kubernetes [Namespace](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/). It is designed to solve access management within multi-user kubernetes cluster.
+
+
+Profile access management provides namespace level isolation based on:
+
+* Kubernetes RBAC
+* Istio AuthorizationPolicy
```

Comment:
I don't this this helps a new user, as this section is trying to introduce the general concept of a Profile without overwhelming the reader:

```suggestion
```